tools/docker/quorum-multi-party-all-in-one/Dockerfile

```diff
@@ -0,0 +1,137 @@
+# And we cannot upgrade to DinD 19 because of
+# https://github.com/docker-library/docker/issues/170
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade "pip>=21" setuptools
+
+# Without this the docker-compose installation crashes, complaining about
+# a lack of rust compiler...
+# RUN pip install setuptools_rust
+ENV CRYPTOGRAPHY_DONT_BUILD_RUST=1
+
+# Install Docker Compose which is a dependency of Fabric Samples
+RUN pip install docker-compose
+
+# Need git to clone the sources of the Fabric Samples repository from GitHub
+RUN apk add --no-cache git
+
+# Fabric Samples needs bash, sh is not good enough here
+RUN apk add --no-cache bash
+
+# The file binary is used to inspect exectubles when debugging container image issues
+RUN apk add --no-cache file
+
+# Need NodeJS tooling for the Typescript contracts
+RUN apk add --no-cache npm nodejs
+
+# Needed because the Fabric binaries need the GNU libc dynamic linker to be executed
+# and alpine does not have that by default
+# @see https://askubuntu.com/a/1035037/1008695
+# @see https://github.com/gliderlabs/docker-alpine/issues/219#issuecomment-254741346
+RUN apk add --no-cache libc6-compat
+
+RUN apk add --no-cache --update chromium
+
+ENV CACTUS_CFG_PATH=/etc/hyperledger/cactus
+RUN mkdir -p $CACTUS_CFG_PATH
+# OpenSSH - need to have it so we can shell in and install/instantiate contracts
+RUN apk add --no-cache openssh augeas
+
+# Configure the OpenSSH server we just installed
+RUN augtool 'set /files/etc/ssh/sshd_config/AuthorizedKeysFile ".ssh/authorized_keys /etc/authorized_keys/%u"'
+RUN augtool 'set /files/etc/ssh/sshd_config/PermitRootLogin yes'
+RUN augtool 'set /files/etc/ssh/sshd_config/PasswordAuthentication no'
+RUN augtool 'set /files/etc/ssh/sshd_config/PermitEmptyPasswords no'
+RUN augtool 'set /files/etc/ssh/sshd_config/Port 22'
+RUN augtool 'set /files/etc/ssh/sshd_config/LogLevel DEBUG2'
+RUN augtool 'set /files/etc/ssh/sshd_config/LoginGraceTime 10'
+# Create the server's key - without this sshd will refuse to start
+RUN ssh-keygen -A
+
+# Generate an RSA keypair on the fly to avoid having to hardcode one in the image
+# which technically does not pose a security threat since this is only a development
+# image, but we do it like this anyway.
+RUN mkdir ~/.ssh
+RUN chmod 700 ~/.ssh/
+RUN touch ~/.ssh/authorized_keys
+RUN ["/bin/bash", "-c", "ssh-keygen -t rsa -N '' -f $CACTUS_CFG_PATH/quorum-aio-image <<< y"]
+RUN mv $CACTUS_CFG_PATH/quorum-aio-image $CACTUS_CFG_PATH/quorum-aio-image.key
+RUN cp $CACTUS_CFG_PATH/quorum-aio-image.pub ~/.ssh/authorized_keys
+
+RUN apk add --no-cache util-linux
+
+# FIXME - make it so that SSHd does not need this to work
+RUN echo "root:$(uuidgen)" | chpasswd
+
+RUN git clone https://github.com/travis-payne/quorum-dev-quickstart.git
+
+WORKDIR /quorum-dev-quickstart
+
+RUN git checkout master
```

Comment:
@travis-payne Recommending to use exact commit hash or tag so that the container build is stable